docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,35 +1,372 @@
+          <li>Your Dagster jobs aren't computationally expensive</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Hybrid</strong>
+      </td>
+      <td>
+        With a Hybrid Deployment, your Dagster code is executed in your
+        infrastructure. The agent runs in your infrastructure, executes your
+        code, and streams metadata about code execution over HTTPS back to
+        Dagster Cloud. Refer to the{" "}
+        <a href="/dagster-cloud/deployment/hybrid">
+          Hybrid deployment documentation
+        </a>{" "}
+        for more info.
+        <br></br>
+        <br></br>
+        Dagster Cloud supports a handful of agents, each with its own prerequisites
+        for setup. Refer to the <a href="/dagster-cloud/deployment/agents">
+          Agent documentation
+        </a> for more info.
+      </td>
+      <td>
+        <ul
+          style={{
+            marginTop: "0px",
+          }}
+        >
+          <li
+            style={{
+              marginTop: "0px",
+            }}
+          >
+            You want to use your own infrastructure
+          </li>
+          <li>You can create and configure an agent in your infrastructure</li>
+          <li>
+            You can use one of the{" "}
+            <a href="/dagster-cloud/deployment/agents">
+              agents currently supported for Dagster Cloud
+            </a>
+          </li>
+        </ul>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+Select the deployment type you want to use and [proceed to the next step](#step-3-deploy-your-code). **Note**: If you find your deployment type isn't right for you, it can be changed at any time.
+
+---
+
+## Step 3: Deploy your code
+
+Now that you've selected a deployment type, the next step is to tell Dagster Cloud the location of your Dagster code. In addition to this, you'll set up Continuous Integration (CI) and Continuous Deployment (CD). Once completed, [Branch Deployments](/dagster-cloud/developing-testing/branch-deployments) and continuous code location deployments will be enabled for your Dagster project.
+
+The steps for accomplishing this vary depending on the **deployment type** you selected in the previous step:
+
+<TabGroup>
+<TabItem name="Select deployment type">
+
+Click the tab for your deployment type - **Serverless** or **Hybrid** - to view what's next.
+
+</TabItem>
+<TabItem name="Serverless">
+
+### Serverless
+
+For **Serverless deployments**, there are two ways to deploy your code to Dagster Cloud:
+
+- **Start from a template** - Use one of our quickstart templates to get up and running. All templates come with CI/CD already configured and will be cloned to a new GitHub repository.
+
+- **Import an existing project** - Import an existing Dagster project using our GitHub app or the [dagster-cloud CLI](/dagster-cloud/developing-testing/dagster-cloud-cli). **Note**: If using the GitHub app, Dagster Cloud will automatically set up CI/CD for you.
```

Comment:
It can be here, but this term is also in the NUX UI. It specifically says `Import an existing project`.

I'll update the description here, but if we have concerns over the term `project` in this context, we may want to discuss UI updates as well